### PR TITLE
PP-5717: Configure adminusers to use json logging

### PR DESCRIPTION
**Pull request by oswaldquek**
## WHAT YOU DID
_A brief description of the pull request:_

## How to test

- How should it be reviewed? 
- What feedback do you need? 
- If manual testing is needed, give suggested testing steps.

### Logging

- [ ] only emit log lines at ERROR level which require immediate attention from a support engineer. These will trigger a zendesk alert.

### Documentation

- [ ] Updated README.md for any of the following ?

* Introduced any new environment variables / removed existing environment variable
* Added new API / updated existing API definition

### CODE REVIEW

File: src/main/resources/config/config.yaml [hunk 1]
```diff
@@ -9,11 +9,11 @@ server:
...
   level: INFO
   appenders:
-    - type: console
+    - type: logstash-console
       threshold: ALL
-      timeZone: UTC
       target: stdout
-      logFormat: "[%d{yyyy-MM-dd HH:mm:ss.SSS}] [thread=%thread] [level=%-5level] [logger=%logger{15}] [requestID=%X{X-Request-Id:-(none)}] - %msg%n"
+      customFields:
+        container: "connector"
```
Comment (stephencdaly) [inline src/main/resources/config/config.yaml]: should be adminusers

Comment (danworth) [inline src/main/resources/config/config.yaml]: shouldn't this be 'adminusers'?

File: src/test/resources/config/test-it-config.yaml [hunk 2]
```diff
@@ -9,11 +9,11 @@ server:
...
   level: INFO
   appenders:
-    - type: console
+    - type: logstash-console
       threshold: ALL
-      timeZone: UTC
       target: stdout
-      logFormat: "[%d{yyyy-MM-dd HH:mm:ss.SSS}] [thread=%thread] [level=%-5level] [logger=%logger{15}] [requestID=%X{X-Request-Id:-(none)}] - %msg%n"
+      customFields:
+        container: "connector"
```
Comment (danworth) [inline src/test/resources/config/test-it-config.yaml]: shouldn't this be `adminusers`?

File: src/main/resources/config/config.yaml (hunk 1 above)
Comment (oswaldquek) [reply src/main/resources/config/config.yaml]: whoops!!

Comment (oswaldquek) [reply src/main/resources/config/config.yaml]: fixed

**Pull request merged**